From martinez.javier@gmail.com  Thu Aug 12 13:10:43 2010
Subject: Staging: spectra: remove duplicate GLOB_VERSION definition
From: Javier Martinez Canillas <martinez.javier@gmail.com>
To: Randy Dunlap <randy.dunlap@oracle.com>, Greg KH <greg@kroah.com>,  devel@driverdev.osuosl.org, David Woodhouse <David.Woodhouse@intel.com>,  kernel-janitors@vger.kernel.org
Date: Sat, 17 Jul 2010 16:39:54 -0400
Message-ID: <1279399194.14049.18.camel@lenovo>

This is the first patch of a patchset that removes all compilations
warnings in staging/spectra.

These patches are a delta from a previous patchset and it assumes that
these three patches all already applied:

Signed-off-by: Javier Martinez Canillas <martinez.javier@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/spectra/ffsport.c |    1 -
 1 file changed, 1 deletion(-)

--- a/drivers/staging/spectra/ffsport.c
+++ b/drivers/staging/spectra/ffsport.c
@@ -114,7 +114,6 @@ u64 GLOB_u64_Remainder(u64 addr, u32 div
 
 #define GLOB_SBD_NAME          "nd"
 #define GLOB_SBD_IRQ_NUM       (29)
-#define GLOB_VERSION		"driver version 20091110"
 
 #define GLOB_SBD_IOCTL_GC                        (0x7701)
 #define GLOB_SBD_IOCTL_WL                        (0x7702)
